### ISSUE TITLE
Word lists next word

### ISSUE THREAD

**Issue by Steve-Miller** [sillsdev/cog#31]
This isn't a bug per se, but a UX issue that's getting to me.

I'm in the input tab, Word lists. I have three varieties set up with hundreds of meanings, and I'm entering words now. I enter a word into a cell and hit tab to go to the next cell for the variety. This works fine; the cell is highlighted. But I can't type in it.

If I am using Excel or LibreOffice Calc, I can tab and type in the next cell. I can't do that with Cog. To edit the cell, I must click on it. This might not seem like much, but after entering a couple dozen words, I have found that it slows me way down, and I keep thinking it should act like a spreadsheet table and it doesn't.

**Comment by ddaspit** [comment]
I agree that this can be a bit tedious. I will certainly look into it. Most users import their data from another source (Excel or WordSurv). The ability to edit words was added primarily to allow the user to correct IPA issues. As a workaround, you create your wordlists in Excel and import it into Cog. You can find directions here: https://github.com/sillsdev/cog/wiki/Inputting-Word-Lists#importing-from-excel.

**Comment by Steve-Miller** [comment]
I actually started out with LibreOffice Calc. But then I would have to figure out how to export with a comma delimited list and learn how to import correctly into Cog. It probably wouldn't be that big of a deal, but it's just more time to learn to learn yet two more things, and that gets old after awhile. Then I found that I could add words directly into Cog without the export/import. Eh, why not? I'm transcribing from a recording that someone else did a dozen years ago, so it makes sense to just add words directly in to Cog rather than use a spreadsheet. I had a thought of transcribing into ELAN first, but then I faced more potential of getting the export wrong. Now that I think about it, I suppose I could have used SayMore as a front end to ELAN. But that add yet one more layer to the mix of things that can go wrong. The simplicity of entering directly into Cog won the day. It's so close to working the way I expected now that I was hoping it wouldn't take much to fix it.
